package.json

```diff
@@ -77,6 +77,7 @@
     "@ethersproject/abi": "^5.7.0",
     "@ethersproject/contracts": "^5.7.0",
     "@lifi/types": "^6.0.0",
+    "@safe-global/safe-apps-sdk": "^7.11.0",
```

Comment:
Should be removed from here as per discussion 🙂

Reply:
Removing